src/sentry/api/endpoints/relay_projectconfigs.py

```diff
@@ -64,9 +64,6 @@ def _post(self, request):
                     orgs = {o.id: o for o in orgs if request.relay.has_org_access(o)}
             else:
                 orgs = {}
-            org_options = {
-                i: OrganizationOption.objects.get_all_values(i) for i in six.iterkeys(orgs)
```

Comment:
I don't understand why we do this. The organization options are already fetched like this as part of the endpoint the first time any option is accesssed.

Reply:
I can only assume that the intention was to load the options in bulk for all organizations in one step. At some point this was a direct database query, since that was faster than accessing the cache (but also put too much load on the database, which is why it was reverted). 

Where exactly is the double-fetch?

Note that this changes timing metrics. In the old implementation the time taken to fetch options was part of `relay_fetch_orgs`, and now it is somewhere hidden within `get_project_config`.

Reply:
The thing is that `get_value` already calls get_all_values the first time it's called: https://github.com/getsentry/sentry/blob/21e6626083a9126b42e490f01b9b3a38ce05977f/src/sentry/models/organizationoption.py#L21 `get_all_values` then caches in-memory.

I *thought* there's actually a double-fetch but that's not actually true.

Let me check about the metric shift. I suspect the cache is so fresh it shouldn't matter.

Reply:
updated so it pre-fetches. `get_all_values` does cache in memory

Reply:
To my understanding, we're never calling `get_value` directly anywhere, in favor of passing in options explicitly. This allows us full control over when the options are fetched.

If you prefer to stop passing options in, could we ensure loading the options in the beginning just like now?

Reply:
I think that's now done, no?